src/components/GlobalPreferences/Notifications/Notifications.js

```diff
@@ -102,31 +115,60 @@ export default function Notifications({
       />
     )
   }
-  if (authState === AUTH_PREVERIFY) {
-    return <NotificationsPreVerify email={email} />
+  switch (authState) {
+    case AUTH_PREVERIFY:
+      return <NotificationsPreVerify email={email} />
+    case AUTH_AUTHENTICATED:
+      return (
+        <ManageNotifications
+          dao={dao}
+          onLogout={handleLogout}
+          apps={apps}
+          email={email}
+          token={token}
+        />
+      )
+    case AUTH_UNAUTHENTICATED:
+      return (
+        <NotificationsLogin
+          dao={dao}
+          authState={authState}
+          onEmailChange={handleEmailChange}
+        />
+      )
+    case AUTH_AUTHENTICATING:
+      return (
+        <NotificationsVerifyBox header="Authenticating">
+          <div
+            css={`
+              display: flex;
+              justify-content: center;
+              align-items: center;
+            `}
+          >
+            <LoadingRing />
+          </div>
+        </NotificationsVerifyBox>
+      )
+    case AUTH_AUTHENTICATION_FAILED:
+      return (
+        <NotificationsVerifyBox header="Authentication Failed">
+          <div>
+            Authentication was unsuccessful.{' '}
+            <ButtonBase
```

Comment:
We can just now use `ButtonText` (although it still needs the font-weight override) :).